### PR TITLE
fix: fit iPad extension page UI

### PR DESCRIPTION
**Pull request by zhouhanseng**

### CODE REVIEW

File: src/utils/theme.ts [hunk 1]
```diff
@@ -24,6 +24,15 @@ const base: ThemeOptions = {
     typography: {
         fontFamily: getFontFamily(),
     },
+    breakpoints: {
+        values: {
+            xs: 0,
+            sm: 600,
+            md: 1112,
```
Comment (zhouhanseng) [inline src/utils/theme.ts:31]: According to https://developer.apple.com/library/archive/documentation/DeviceInformation/Reference/iOSDeviceCompatibility/Displays/Displays.html, the largest width of iPad is 1112.

File: src/extension/options-page/DashboardDialogs/Base.tsx [hunk 2]
```diff
@@ -149,6 +149,7 @@ const useDashboardDialogWrapperStyles = makeStyles((theme) =>
             flexDirection: 'column',
             width: (props) => (props.size === 'small' ? 280 : 440),
             padding: (props) => (props.size === 'small' ? '40px 24px !important' : '40px 36px !important'),
+            margin: '0 auto',
```
Comment (zhouhanseng) [inline src/extension/options-page/DashboardDialogs/Base.tsx:152]: image

to

image

Comment (septs) [reply src/extension/options-page/DashboardDialogs/Base.tsx:152]: if can, plz try use flex.

see https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Comment (zhouhanseng) [reply src/extension/options-page/DashboardDialogs/Base.tsx:152]: I know flex, but it seems not easy to do at a glance, I will check again.

Comment (zhouhanseng) [reply src/extension/options-page/DashboardDialogs/Base.tsx:152]: I tried to set its parent to a flex box, but it doesn't looks well:

image

image

I found that its parent has property `'position': 'fix'`, then I get the answer https://stackoverflow.com/a/29036585/4803321

> Absolutely positioned boxes are **taken out of the normal flow**.

Comment (septs) [reply src/extension/options-page/DashboardDialogs/Base.tsx:152]: OK.

**Review by septs — CHANGES_REQUESTED** [review]
image

**Comment by zhouhanseng** [comment]
Locally run `./node_modules/.bin/prettier --check ./`, it seems not caused by this PR?

File: webpack.config.ts [hunk 3]
```diff
@@ -37,9 +37,7 @@ export default function (_argvEnv, argv: any) {
     const env: 'production' | 'development' = argv.mode
     const dist = env === 'production' ? src('./build') : src('./dist')
 
-    const enableHMR =
-        env === 'development' &&
-        !Boolean(process.env.NO_HMR)
+    const enableHMR = env === 'development' && !Boolean(process.env.NO_HMR)
```
Comment (zhouhanseng) [inline webpack.config.ts:40]: Fix a prettier format issue.

**Review by Jack-Works — APPROVED** [review]

**Review by septs — APPROVED** [review]

**Pull request merged**